test/inference/gibbs.jl

```diff
@@ -49,7 +49,7 @@ include(dir*"/test/test_utils/AllUtils.jl")
         alg = Gibbs(
             MH(:s),
             HMC(0.2, 4, :m))
-        chain = sample(gdemo(1.5, 2.0), alg, 3000)
+        chain = sample(gdemo(1.5, 2.0), alg, 5000)
```

Comment:
I'm still surprised that we have to increase the number of samples for all these HMC changes (also in the previous PR) - shouldn't a hopefully better initialization improve convergence?

Reply:
Not necessarily. The initialization is just made consistent with Stan. That doesn't necessarily make it better all the time. Also there is randomness involved and the tolerance was not satisfied by a tiny amount so it's very much seed-sensitive.

Reply:
Although I agree with Mohamed that it's not necessary better, I'm also superise that we need more iterations.